builder/proxmox/config.go

```diff
@@ -183,6 +191,31 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 			errs = packer.MultiErrorAppend(errs, fmt.Errorf("disk format must be specified for pool type %q", c.Disks[idx].StoragePoolType))
 		}
 	}
+	for idx := range c.AdditionalISOFiles {
+		if c.AdditionalISOFiles[idx].Device == "" {
+			log.Printf("AdditionalISOFile %d Device not set, using default 'ide'", idx)
+			c.AdditionalISOFiles[idx].Device = "ide"
+		}
+		if !contains([]string{"ide", "sata", "scsi"}, c.AdditionalISOFiles[idx].Device) {
+			errs = packer.MultiErrorAppend(errs, fmt.Errorf("%q is not a valid AdditionalISOFile Device", c.AdditionalISOFiles[idx].Device))
+		}
+		if c.AdditionalISOFiles[idx].BusNumber == 0 {
+			log.Printf("AdditionalISOFile %d number not set, using default: '3'", idx)
+			c.AdditionalISOFiles[idx].BusNumber = 3
+		}
```

Comment:
If I leave multiple additional isos without a bus number, they will overwrite each other this way. I can see three options:
1. Make bus_number mandatory (and in that case, maybe merge it with device_type so you'd specify the entire device like `device="ide3"`)
2. Keep track of what device numbers we've used and use the next unused one. This will probably get tricky.
3. Error out if there's more than one element without a set number.

I'd lean towards option 1 because it will more obvious, although it does sacrifice a small amount of convenience when just adding a single additional iso.